docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,210 @@
+
+#### Streamlined Kubernetes diagnostics 
+
+New PMM Client Docker images now include essential troubleshooting tools:
+
+- `tar` enables easier file transfer in and out of containers using the `kubectl cp` command.
+- `curl` allows direct checking of exporters to verify their proper functioning.
+
+### Improved security
+
+This update addresses several security issues related to the GNU C Library (Glibc), specifically affecting the Name Service Cache Daemon (nscd). Additionally, it resolves multiple security vulnerabilities by upgrading various third-party packages.
+
+### Important changes to PMM Update process
+
+To ensure you can access the most up-to-date and secure versions, we've implemented two key changes to the update process:
+
+#### Transition to a dedicated repository
+
+Since July 1st 2024, PMM updates are distributed exclusively through the dedicated `repo.percona.com/pmm2-client` repository. 
+
+To smoothly transition to this repository and verify that your system is correctly configured, download and run our automated script from GitHub: [check_percona_packages.py](https://raw.githubusercontent.com/Percona-Lab/release-aux/main/scripts/check_percona_packages.py).
+
+For detailed instructions and more information about this change, check out [our recent blog post](https://www.percona.com/blog/ensure-the-correct-repositories-are-enabled-for-percona-packages/) and [updated installation instructions](../setting-up/index.md).
+
+#### Update process for older PMM versions
+
+With the migration of PMM’s base image from CentOS 7 to Oracle Linux 9 last year, users running PMM version 2.38 or earlier may experience issues when updating PMM to the latest version using the UI **Update** button.
+
+To avoid these issues and ensure you continue receiving updates:
+
+- **For Docker instances**: perform a one-time Docker update to migrate your installation to the Oracle Linux 9-based image.
+- **For Non-Docker Instances (OVF/AMI)**: follow the upgrade instructions outlined in our [PMM Modernization blog post](https://www.percona.com/blog/pmm-modernized-enterprise-grade-foundation).
+
+### Experimental ARM support for PMM Client
+
+PMM 2.43 introduces experimental support for ARM architecture in PMM Clients, responding to the growing adoption of ARM in data centers and cloud environments. 
+
+For this, PMM now includes pre-built ARM binaries for key features like `node_exporter` and `mysqld_exporter`, enabling monitoring of ARM-based infrastructure while maintaining compatibility with existing PMM Server installations.
+
+As an experimental feature, ARM support will be refined and stabilized in future releases, paving the way for unified monitoring across x86 and ARM architectures.
```

Comment:
Call to action for the user: give feedback via forums to help expedite the path to GA.